### PR TITLE
Update webpack to v5, adjust config

### PR DESCRIPTION
**Pull request by nl0**

### CODE REVIEW

File: catalog/package.json [hunk 1]
```diff
@@ -44,11 +44,7 @@
     "test:watch": "npm run test:only -- --watch",
     "prettify": "prettier --write"
   },
-  "browserslist": [
-    "last 2 versions",
-    "> 1%",
-    "IE 10"
-  ],
+  "browserslist": ["last 2 versions", "> 1%"],
```
Comment (fiskus) [inline catalog/package.json:47]: calling `npm install` formats this array to multiline:

```diff
-  "browserslist": ["last 2 versions", "> 1%"],
+  "browserslist": [
+    "last 2 versions",
+    "> 1%"
+  ],
```

Linux, node v15.5.0, npm 7.3.0

File: catalog/internals/webpack/webpack.prod.babel.js [hunk 2]
```diff
@@ -147,4 +44,20 @@ module.exports = require('./webpack.base.babel')({
   performance: {
     assetFilter: (assetFilename) => !/(\.map$)|(^(main\.|favicon\.))/.test(assetFilename),
   },
+
+  // options passed to all instances of HtmlWebpackPlugin
+  html: {
+    minify: {
```
Comment (fiskus) [inline catalog/internals/webpack/webpack.prod.babel.js:50]: `build` directory has the same size with just `minify: true`. I don't think this fine-tuning worth it

**Review by fiskus — APPROVED** [review]

File: catalog/package.json (hunk 1 above)
Comment (nl0) [reply catalog/package.json:47]:
```suggestion
  "browserslist": [
    "last 2 versions",
    "> 1%"
  ],
```

File: catalog/internals/webpack/webpack.prod.babel.js (hunk 2 above)
Comment (nl0) [reply catalog/internals/webpack/webpack.prod.babel.js]: looks like we can omit this altogether and it will still minify the output in production mode

Comment (fiskus) [reply catalog/internals/webpack/webpack.prod.babel.js]: :heavy_plus_sign:

**Pull request merged**